cmd/infracost/comment.go

```diff
@@ -58,8 +58,8 @@ func commentCmd(ctx *config.RunContext) *cobra.Command {
 		_ = subCmd.Flags().MarkHidden("show-changed")
 		subCmd.Flags().Bool("skip-no-diff", false, "Skip posting comment if there are no resource changes. Only applies to update, hide-and-new, and delete-and-new behaviors")
 		_ = subCmd.Flags().MarkHidden("skip-no-diff")
-		subCmd.Flags().String("guardrail-check-path", "", "Path to Infracost guardrail data (experimental)")
-		_ = subCmd.Flags().MarkHidden("guardrail-check-path")
+		subCmd.Flags().String("additional-path", "", "Path to additional comment text (experimental)")
```

Comment:
```suggestion
		subCmd.Flags().String("additional-comment-data-path", "", "Path to additional comment text (experimental)")
```

Might be a bit more readable, but I guess it's not too much of an issue as this is internal

Reply:
yeah I was hoping you would have a better idea for the name